docs/application/dotnet/guides/user-interface/nui/text.md

```diff
@@ -756,6 +756,106 @@ The following table lists the available `TextEditor` properties:
 | `TranslatableText`                 | String      | Specifies the `TranslatableText` property that sets the SID value. |
 
 
+### Use public APIs copy, cut, paste and select text in TextField or TextEditor
+
+A set of APIs that provide developers with the functionality of copy, cut, paste & select text.
+A sample code on how to benefit from them is shown below.
+
+Developers can define an action for users by writing the code below.
+I.e. when the user clicks on a button.
+
+The following code is an example to copy and cut the text from source to destination text fields:
+
+```csharp
+// Create two TextField
+var textFieldSrc = new TextField(); // Source TextField
+textFieldSrc.Text = "Welcome! This is a great world!";
+
+var textFieldDest = new TextField(); // Destination TextField
+textFieldDest.Text = "";
+
+```
+
+`SelectText` is an API that takes two parameters, the start index and the end index.
+This API selects text based on the start and end index declared by the user.
+The following code shows how to use `SelectText` on `TextField`:
+
+```csharp
+// Step 01: Select text from source text filed
+textFieldSrc.SelectText (9,15); //"This is" is the selected text
+
+ // At this point:
+ // The TEXT in textFieldSrc is "Welcome! This is a great world!"
+ // The TEXT in textFieldDest is ""
+
+```
+
+`TextUtils.copyToClipboard` is an API that takes one parameter, either TextField or TextEditor.
+Using this API, you can copy the text into the clipboard that was selected based on the declared indexes in the previous code.
+The following code shows how to use `CopyToClipboard` with `TextField`:
+
+```csharp
+// Step 02: Copy text from source text filed
+ string copiedText = TextUtils.CopyToClipboard(textFieldSrc);
+
+ // At this point:
+ // The TEXT in textFieldSrc is "Welcome! This is a great world!"
+ // The TEXT in textFieldDest is ""
+
+```
+
+`TextUtils.pasteTo` is an API that takes one parameter, either TextField or TextEditor.
+Using this API, you can paste the most recently copied/cut text item from the clipboard into the text control.
+
+   > [!NOTE]
+   > The pasted text will be inserted in the current cursor position and if the text control has no focus, the text will be appended to the last cursor position and the text control will gain focus. If some text inside the text control is selected, it will be replaced by the pasted text.
+
+
+The following code shows how to use `PasteTo` with `TextField`:
+
+```csharp
+// Step 03: Paste text into destination text filed
+ TextUtils.PasteTo(textFieldDest);
+
+ // At this point:
+ // The TEXT in textFieldSrc is "Welcome! This is a great world!"
+ // The TEXT in textFieldDest is "This is"
+
+```
+
+`TextUtils.cutToClipboard` is an API that takes one parameter, either TextField or TextEditor.
```

Comment:
Please revert this change to `TextUtils.CutToClipboard`, I had a misconception about the API.
```suggestion
`TextUtils.CutToClipboard` is an API that takes one parameter, either TextField or TextEditor.
```

Reply:
Done.